akka-serialization-jackson/src/main/resources/reference.conf

```diff
@@ -9,8 +9,6 @@
 akka.serialization.jackson {
 
   # The Jackson JSON serializer will register these modules.
-  # It is also possible to use jackson-modules = ["*"] to dynamically
-  # find and register all modules in the classpath.
```

Comment:
removed this feature because it's questionable if it's useful (too magic) and it's very difficult to test